internal/gen.go

```diff
@@ -94,6 +114,7 @@ func Generate(ctx context.Context, req *plugin.GenerateRequest) (*plugin.Generat
 
 	var queries core.Queries = make(map[string][]core.Query)
 	var embeddedModels core.EmbeddedModels = make(map[string][]core.QueryReturn)
+	var nullableHelpers core.NullableHelpers = core.NullableHelpers{}
```

Comment:
You don't need to specify the type here, just `nullableHelpers := core.NullableHelpers{}` would be fine

Reply:
Removed :)